app/lib/data_type_validator.rb

```diff
@@ -39,7 +39,7 @@ def is_email?(value)
   end
 
   def is_url?(value)
-    value.match?(/^((ftp|http|https):\/\/)?[a-zA-Z0-9]+(.[a-zA-Z0-9])?(:[0-9]+)?/)
+    value.match?(/^((ftp|http|https):\/\/)?[a-zA-Z0-9]+((\.[a-zA-Z0-9]+)|(:[0-9]+)+)/)
```

Comment:
Agreed. I'm fixing this part of the URL and adding an optional path regex after it (updated from the current path regex) to ensure that the URL is fully valid.